decidim-core/lib/decidim/form_builder.rb

```diff
@@ -874,6 +874,7 @@ def language_selector_select(locales, tabs_id, name)
         content_tag(:select, id: tabs_id, class: "language-change") do
           locales.each_with_index.inject("".html_safe) do |string, (locale, index)|
             title = I18n.with_locale(locale) { I18n.t("name", scope: "locale") }
+            title += " (error!)" if error?(name_with_locale(name, locale))
```

Comment:
If wee go with this solution, the `(error!)` part should be using the `I18n` system! 😄

Reply:
Something like

```ruby
title = if error?(name_with_locale(name, locale))
  I18n.with_locale(locale) { I18n.t("name_with_error", scope: "locale") }
else
  I18n.with_locale(locale) { I18n.t("name", scope: "locale") } 
end
```